### PR TITLE
Fix the New York household credit base and additional amount brackets

### LINKED ISSUE #4399: Fix New York household credit calculation
Using the 0.736.0 version, the `ny_household_credit` variable seems to be miscalculated in some cases.   Consider the following simple test (which fails because PEUS calculates a credit of $110):

```
- name: Simple ny_household_credit test
  absolute_error_margin: 0.01
  period: 2021
  input:
    people:
      person1:
        age: 40
      person2:
        age: 10
      person3:
        age: 11
      person4:
        age: 13
      person5:
        age: 15
    tax_units:
      tax_unit:
        members: [person1, person2, person3, person4, person5]
        adjusted_gross_income: 20_020
    spm_units:
      spm_unit:
        members: [person1, person2, person3, person4, person5]
    households:
      household:
        members: [person1, person2, person3, person4, person5]
        state_fips: 36  # NY
  output:  # from 2021 NY IT-213 instructions lookup table on page 20
    ny_household_credit: 100
```

The expected result is obtained from this table:

NY-hh-credit

### PR DESCRIPTION
**Pull request by Shinuing**
Fixes #4399

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ny/tax/income/credits/household_credit/non_single/base.yaml [hunk 1]
```diff
@@ -28,6 +28,10 @@ brackets:
       1986-01-01: 60
   - threshold:
       1986-01-01: 20_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ny/tax/income/credits/household_credit/non_single/base.yaml:30]:
```suggestion
      1986-01-01: 22_000
```

File: policyengine_us/parameters/gov/states/ny/tax/income/credits/household_credit/non_single/base.yaml [hunk 2]
```diff
@@ -28,6 +28,10 @@ brackets:
...
+    amount:
+      1986-01-01: 60
+  - threshold:
+      1986-01-01: 22_000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ny/tax/income/credits/household_credit/non_single/base.yaml:34]:
```suggestion
```

File: policyengine_us/parameters/gov/states/ny/tax/income/credits/household_credit/non_single/base.yaml (hunk 1 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ny/tax/income/credits/household_credit/non_single/base.yaml:30]: Would something like this work? (We are basically extending the $60 bracket right?)

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Comment by Shinuing** [comment]
Not really. The additional amount are different.


On Apr 17, 2024, at 17:27, Pavel Makarchuk ***@***.***> wrote:

﻿

@PavelMakarchuk requested changes on this pull request.

________________________________

In policyengine_us/parameters/gov/states/ny/tax/income/credits/household_credit/non_single/base.yaml<https://github.com/PolicyEngine/policyengine-us/pull/4414#discussion_r1569616343>:

> @@ -28,6 +28,10 @@ brackets:
       1986-01-01: 60
   - threshold:
       1986-01-01: 20_000


⬇️ Suggested change

-      1986-01-01: 20_000
+      1986-01-01: 22_000


________________________________

In policyengine_us/parameters/gov/states/ny/tax/income/credits/household_credit/non_single/base.yaml<https://github.com/PolicyEngine/policyengine-us/pull/4414#discussion_r1569616461>:

> +    amount:
+      1986-01-01: 60
+  - threshold:
+      1986-01-01: 22_000


⬇️ Suggested change

-    amount:
-      1986-01-01: 60
-  - threshold:
-      1986-01-01: 22_000


________________________________

In policyengine_us/parameters/gov/states/ny/tax/income/credits/household_credit/non_single/base.yaml<https://github.com/PolicyEngine/policyengine-us/pull/4414#discussion_r1569616737>:

> @@ -28,6 +28,10 @@ brackets:
       1986-01-01: 60
   - threshold:
       1986-01-01: 20_000


Would something like this work? (We are basically extending the $60 bracket right?)

**Comment by PavelMakarchuk** [comment]
@Shinuing is it $60 between 7,000 and 20,000 for the base amount?

**Comment by Shinuing** [comment]
> @Shinuing is it $60 between 7,000 and 20,000 for the base amount?

No it's $65

Screen Shot 2024-04-17 at 8 14 21 PM

**Comment by PavelMakarchuk** [comment]
> > @Shinuing is it $60 between 7,000 and 20,000 for the base amount?
> 
> No it's $65
> 
> Screen Shot 2024-04-17 at 8 14 21 PM

My bad for the misunderstanding, cant we summarize this bracket into one, (just in the particular file that we are talking about):
image

**Comment by Shinuing** [comment]
> My bad for the misunderstanding, cant we summarize this bracket into one, (just in the particular file that we are talking about): 

NO! The additional amount are different

**Comment by PavelMakarchuk** [comment]
> > My bad for the misunderstanding, cant we summarize this bracket into one, (just in the particular file that we are talking about):
> 
> NO! The additional amount are different

But isn't this represented in base.yaml?

**Comment by Shinuing** [comment]
> > > My bad for the misunderstanding, cant we summarize this bracket into one, (just in the particular file that we are talking about):
> > 
> > 
> > NO! The additional amount are different
> 
> But isn't this represented in base.yaml?

Yes you're right. Just merged

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**